# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1245856444 14400
#      Wed Jun 24 11:14:04 2009 -0400
# Node ID d7f45ff0797ba1e4dba9e046f60e26394a6e8407
# Parent  5497b0795fb96ce737c21cd7adcbe0fb99aa3abc
add Alexander Mamonov to contributors list

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -126,16 +126,17 @@ Hoxide Ma
 James Macnicol
 Jens-Uwe Mager
 Ricardo Marranita
 Orestes Mas
 Makoto Matsumoto
 Tatsuro Matsuoka
 Laurent Mazet
 G. D. McBain
+Alexander Mamonov
 Christoph Mayer
 Thorsten Meyer
 Petr Mikulik
 Stefan Monnier
 Antoine Moreau
 Kai P. Mueller
 Victor Munoz
 Carmen Navarrete
